// @powerjavascript/power.p.js/src/color.p.mjs
import chalk from 'chalk';

const reset = chalk.reset;
const bold = chalk.bold;
const dim = chalk.dim;
const italic = chalk.italic;
const underline = chalk.underline;
const inverse = chalk.inverse;
const hidden = chalk.hidden;
const strikethrough = chalk.strikethrough;

const black = chalk.black;
const red = chalk.red;
const green = chalk.green;
const yellow = chalk.yellow;
const blue = chalk.blue;
const magenta = chalk.magenta;
const cyan = chalk.cyan;
const white = chalk.white;
const gray = chalk.gray;
const grey = chalk.grey; 

const bgBlack = chalk.bgBlack;
const bgRed = chalk.bgRed;
const bgGreen = chalk.bgGreen;
const bgYellow = chalk.bgYellow;
const bgBlue = chalk.bgBlue;
const bgMagenta = chalk.bgMagenta;
const bgCyan = chalk.bgCyan;
const bgWhite = chalk.bgWhite;

export {
  reset,
  bold,
  dim,
  italic,
  underline,
  inverse,
  hidden,
  strikethrough,
  black,
  red,
  green,
  yellow,
  blue,
  magenta,
  cyan,
  white,
  gray,
  grey,
  bgBlack,
  bgRed,
  bgGreen,
  bgYellow,
  bgBlue,
  bgMagenta,
  bgCyan,
  bgWhite
};
